docs/examples/basic_meter/calling_conventions.py

```diff
@@ -53,11 +45,18 @@
     unit="1",
     value_type=int,
     metric_type=Counter,
-    label_keys=("environment",),
 )
 
 labels = {"environment": "staging"}
 
+# Views are used to define an aggregation type to use for a specific metric
+counter_view = View(requests_counter, CountAggregation())
+clicks_view = View(clicks_counter, CountAggregation())
+
+# Register the views to the view manager to use the views
+meter.register_view(counter_view)
+meter.register_view(clicks_view)
```

Comment:
qn on `requests_counter`. This counter instrument is created much before a view involving it is created and registered. What would happen to the `requests_counter.add()` calls - how would they be aggregated, before the `register_view` call occurred? 

Should we require that all views must be registered beforehand?

Reply:
> Should we require that all views must be registered beforehand?
Good question. Yes I believe we should enforce this as a requirement.

Reply:
Dynamic view management would certainly be more challenging. Let's leave it out for now.